### PR TITLE
Binary Search Tree in C

### PR DESCRIPTION
**Pull request by VishalS-HK**
Binary Search Tree: 
i)Bst is a type of bt where in value of lst is lower than value of temp(root).
value of Rst is higher than temp value.
ii)Operations to be performed on bst are:
   a)create
   b)insert
   c)delete
   d)search

### CODE REVIEW

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Kindly update the **README.md**

**Comment by VishalS-HK** [comment]
Done updating the readme.md

File: README.md [hunk 1]
```diff
@@ -52,7 +52,8 @@
 | 15.  | Selection Sort | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Sorting/selection_sort.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Sorting/selection_sort.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Sorting/selection_sort.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Sorting/selection_sort.py) | Sorting |
 | 16.  | AVL Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/AVLTree.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/AVLTree.py) | Trees |
 | 17.  | Red-Black Tree | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Trees/red_black.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Trees/red_black.py) | Trees |
-| 18.  | Decision Tree | [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Machine%20Learning/Decision%20Tree/decision_tree.py) | Machine Learning |
+| 18.  | Binary Search Tree | [C](https://github.com/Kumar-laxmi/Algorithms/tree/main/C/Trees) | Trees |
```
Comment (Kumar-laxmi) [inline README.md:55]: Please put Binary Seach Tree between AVL Tree and Red-Black Tree

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

**Comment by VishalS-HK** [comment]
Done

**Review by Kumar-laxmi — APPROVED** [review]

**Comment by Kumar-laxmi** [comment]
@VishalS-HK Congratulations!, kindly star this repo and follow me for more updates

**Pull request merged**